src/client/app/containers/editor.ts

```diff
@@ -39,6 +39,17 @@ export class Editor implements AfterViewInit {
      */
     async ngAfterViewInit() {
         this._monacoEditor = await this._monaco.create(this._editor, { theme: 'vs' });
+        let editor = this._monacoEditor;
+        editor.addAction({
+            id: 'trigger-suggest',
```

Comment:
could you add a comment:

        id: 'trigger-suggest' /* must be unique id */